www-servers/pound/files/pound.init-1.9

```diff
@@ -1,5 +1,5 @@
 #!/sbin/openrc-run
-# Copyright 1999-2004 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
I don't know why you're making this change, as nothing else is touched in this file.

Reply:
I haven't changed it, repoman did it